spyder_kernels/console/start.py

```diff
@@ -16,6 +16,15 @@
 import sys
 import site
 
+# Remove current directory from sys.path to prevent kernel
+# crashes when people name Python files or modules with
+# the same name as standard library modules.
```

Comment:
```suggestion
# Remove current directory from sys.path to prevent kernel crashes when people
# name Python files or modules with the same name as standard library modules.
```

Take advantage of the wider space at this position.